sophisticated-business/functions.php

```diff
@@ -34,38 +34,6 @@ function sophisticated_business_setup() {
 			'header-text' => array( 'site-title' ),
 		)
 	);
-
```

Comment:
This was added relatively recently, maybe @allancole or @scruffian can tell us if this should live on SVN?:

https://github.com/Automattic/themes/commit/0443d627c7a335531337e5c8af4aae6563607e41#diff-6159e326e34c8c1cd265369bf123f8992cf12dd3948ad6f7f5b38d2f2ab9d579

Reply:
Comparing this SVN history: https://opengrok.a8c.com/source/history/pub/sophisticated-business/functions.php
To this GIT history: https://github.com/Automattic/themes/commits/trunk/sophisticated-business/functions.php

It looks like those changes were made but never deployed. 

As much as I don't want to lose work I will recommend that we bring in this change, clobbering the work done represented here, otherwise it will be changing the theme that is in production.  I believe that if it warrants bringing in this change that we should do so AFTER the consolidation.